api/src/main/java/org/pmiops/workbench/audit/ActionAuditEvent.kt

```diff
@@ -0,0 +1,22 @@
+package org.pmiops.workbench.audit
+
+import java.util.UUID.randomUUID
+
+class ActionAuditEvent(
+    val timestamp: Long,
+    val agentType: AgentType,
+    val agentId: Long,
+    val agentEmailMaybe: String?,
+    val actionId: String,
+    val actionType: ActionType,
+    val targetType: TargetType,
+    val targetPropertyMaybe: String? = null,
+    val targetIdMaybe: Long? = null,
+    val previousValueMaybe: String? = null,
+    val newValueMaybe: String? = null
```

Comment:
I always used to name my optionals with maybe, but that hasn't caught on here. Kotlin will remind you at compile time if you forget I suppose, so I guess I should embrace it.

I'd like to do that rename on my follow-on branch to avoid clashes though.